### PR TITLE
Add GH workflows, templates etc.

### PR DESCRIPTION
**Pull request by hariso**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 .PHONY: build-local build-noop-dest run-local run-latest run-latest-nightly print-results
 
+build:
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build main.go
```
Comment (lovromazgon) [inline Makefile:4]: Hmm, why are we building for linux here? I understand it for the plugin as we expect it to run inside a docker container, but this tool will run on the host, right?

File: noopdest/main.go [hunk 2]
```diff
@@ -16,9 +16,11 @@ package noopdest
 
 import (
 	"context"
+
 	sdk "github.com/conduitio/conduit-connector-sdk"
 )
 
+//nolint:deadcode,unused //entry point for the plugin, when Conduit runs it
```
[the remark this answers is not in the document]
Comment (hariso) [reply noopdest/main.go:23]: Yup, you're right there. It's fixed in the other PR #7.

File: Makefile (hunk 1 above)
Comment (hariso) [reply Makefile:4]: Good point. I've copy pasted it from the other command, but it's current usage is only in the GitHub workflow, which is on Linux (to make sure it builds). I'll fix.

File: .github/ISSUE_TEMPLATE/1-feature-request.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: Feature Request
```
[the remark this answers is not in the document]
Comment (hariso) [reply .github/ISSUE_TEMPLATE/1-feature-request.yml:1]: I've copied this from the file connector, looks like it's missing there too.

**Review by lovromazgon — APPROVED** [review]

**Pull request merged**